.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
@SrinivasanTarget like I said, I don't have a strong preference here :) We can go with npm, it will indeed keep repo a bit cleaner as well. For me it's fine to merge it with npm solution.